CONTRIBUTING.md

```diff
@@ -185,16 +185,30 @@ Here is the svg for the Adobe Photoshop icon as an example:
 
 Icon metadata should be added to the `_data/simple-icons.json` file. Each icon in the array has three required values:
 
-  * The `title` of the new SVG.
-  * A `hex` color value that matches the brand's primary color. All uppercase and without the `#` pound symbol.)
-  * The `source` URL of the logo being used. There are [more details below](#source-guidelines).
+* The `title` of the new SVG.
+* A `hex` color value that matches the brand's primary color. All uppercase and without the `#` symbol.
+* The `source` URL of the logo being used. There are [more details below](#source-guidelines).
 
-Additionally, there are also optional fields that may provided for an icon:
```

Comment:
I'd be in favour of leaving this line in and linking it to the relevant section so that those contributing an icon don't miss it.